### DISCUSSION TITLE
[Esame 23 Giugno 2020] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#240] [Esami - Bertini @ Informatica]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/cdfb9fde-beab-4824-8d66-cf821feea4bf)

**Comment by LeoCosta02** [comment]
uguale a foglio6 es 3

- #43
